included/4.0.2/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# WARNING: this file was autogenerated by generate-included-image.js
+# using
+#   npm run add:included -- 4.0.2 --from
```

Comment:
@flotwig how are you making these images?!!! Why is the `--from` blank here? If you run `npm run add:included` it should throw an error if `--from` is empty